distance-calculator/calculator/src/main/java/io/pravega/flinkapp/InfluxdbSink.java

```diff
@@ -43,7 +44,23 @@ public void invoke(OutSenorData value) {
     }
 
     @Override
-    public void open(Configuration config) {
+    public void open(Configuration config) throws IOException {
+        String flinkfilepath = getEnvVar("FLINK_CONF_DIR", "/etc/flink");
+        flinkfilepath = flinkfilepath+"/"+"flink-conf.yaml";
+        System.out.println("flink conf dir " + flinkfilepath);
+            ParameterTool params = ParameterTool.fromPropertiesFile(flinkfilepath);
+            influxdbDbName = params.get("metrics.reporter.influxdb.db","test_flnk");
```

Comment:
Make sure to include the chart for the creation of the influxdb. Otherwise other uses won't be able to recover this demo.

Reply:
We are not creating any InfluxDB on our own , we are reusing the metrics influxdb created as part of Flink Cluster . So , for this demo code , InfluxDB is treated as blackbox for user.

Reply:
ok, if that is the case, may consider to add the helm charts for flink cluster, flink app, grafanadashboard. Meanwhile, pravega-gateway also needs to be upgraded and to supported to written into SDP pravega.

Reply:
the helm chart is worked upon and will be adding cluster , app and grafana components to it .. Pravega-gateway seemed redundant now as "Pravega-MQTT-Broker" has gateway functionality to manage stream and add data into it.

Reply:
I see. Then may delete the pravega-gateway folder and add the instruction on README to ask user create the project with mqtt broker enabled. And in the later helm chart, may have the one for the SDP project also.

Reply:
Done . README has already removed the instruction involving pravega-gateway . It has link to code-hub blog posts so that the stepwise instructions can be referred from there once it gets updated to latest changes.